static/styles/lib/base.scss

```diff
@@ -304,3 +304,16 @@ form.inline {
 .position-absolute{
 	position: absolute !important;
 }
+
+//the following adds fa-share-square-o for each external link except images
+.section-course .content-block{
+	a:not( [href*='https://schul-cloud.org'] ):not( [href^='#'] ):not( [href^='/'] ){
+		&:not( [href$='.jpg'] ):not( [href$='.png'] ):not( [href$='.jpeg'] ):not( [href$='.gif'] ){
```

Comment:
Can we embed .tif and .bmp into courses? If so, add this here. If not, it's fine.

Reply:
I added .tif and .bmp, but i don't know if we will need it

Reply:
Thats why I asked if they even are embedable in courses. If not, we dont need it. I will check it...